ingestion/src/metadata/ingestion/source/database/datalake/connection.py

```diff
@@ -135,7 +135,12 @@ def test_connection(
             func = connection.client.list_buckets
 
     if isinstance(config, AzureConfig):
-        func = partial(connection.client.list_containers, name_starts_with="")
+
+        def my_func(connection, name_starts_with=""):  # pylint: disable=W0613
```

Comment:
name_starts_with is common, let's drop it and send it to the list_containers as is as the arg doesn't do much